### DISCUSSION TITLE
Cant make colour changes

### DISCUSSION THREAD

**Discussion by kkatee** [Aris-t2/CustomCSSforFx#582] [General]
So I had to update firefox today, which I generally avoid so I don't have to spend 2 hours getting the custom css just right again.

Almost everything is right, but I wanted to make some tweaks to the colours, but nothing I change seems to have any effect.

I have been editing the colours in config\color_vairables.css and in css\tabs\classic_squared_tabs.css and nothing I do seems to make any effect on the browser.

**Comment by Aris-t2** [comment]
We need more info here.

Have you tried an clean setup of this projects files? Is this projects default configuration showing up?

Post some screenshots.

Continue here: https://github.com/Aris-t2/CustomCSSforFx/issues/586
